.github/actions/setup-sentry/action.yml

```diff
@@ -138,3 +142,14 @@ runs:
         fi
 
         docker ps -a
+
+    - name: Wait for postgres
+      shell: bash
+      env:
+        WAIT_FOR_POSTGRES: ${{ inputs.wait-for-postgres }}
+      run: |
+        if [ "$WAIT_FOR_POSTGRES" = "true" ]; then
+          until docker run --rm --link sentry_postgres:pg --net sentry postgres:9.6-alpine pg_isready -U postgres -h pg; do
```

Comment:
@joshuarli should I wait for your impl here?

Reply:
Yeah, if you don't mind waiting. https://github.com/getsentry/sentry/pull/21741